torch_tvm/operators.cpp

```diff
@@ -215,6 +219,67 @@ RegisterTVMOperator reg({
        }
        return out;
      }},
+    {Symbol::fromQualString("aten::layer_norm"),
+     [](Node* node, tvm::Array<tvm::relay::Expr> inputs) -> tvm::relay::Expr {
+       auto op = tvm::relay::Op::Get("nn.custom_layer_norm");
+       TORCH_CHECK(
+           inputs.size() == 6,
+           "layer_norm received ",
+           inputs.size(),
+           " inputs");
+       auto normalized_axis_shape = relayToArray<tvm::Integer>(inputs[1]);
+       std::vector<int64_t> shape;
+       shape.reserve(normalized_axis_shape.size());
+       for (auto& dim : normalized_axis_shape) {
+         shape.push_back(dim);
+       }
+       auto attrs = tvm::make_node<tvm::relay::CustomLayerNormAttrs>();
+       attrs->num_axis_to_normalize = normalized_axis_shape.size();
+       auto eps = static_cast<double>(relayToConstant<float>(inputs[4]));
+       attrs->eps = eps;
+
+       TVMContext ctx_;
+       ctx_.device_type = kDLCPU;
+       ctx_.device_id = 0;
+
+       auto val = node->input(0);
+       auto pt_t = val->type()->cast<CompleteTensorType>();
```

Comment:
`inputs[0]` has all this information, right? we should avoid using `Node*` if we can.  This is replicating the logic in `compiler.cpp` and introducing a new API (`scalarTypeToTVMType`)

Reply:
@bwasti, from what I understand input[0] which is Value*, does not have the scalarType of tensor in it. That is why it has TypePtr. I am open to suggestions but I dont see why exposing scalarTypeToTVMType is an issue.